common/src/Assets/EntityModel.cpp

```diff
@@ -202,13 +203,13 @@ namespace TrenchBroom {
         }
 
         void EntityModel::Surface::addIndexedMesh(LoadedFrame& frame, const VertexList& vertices, const Indices& indices) {
-            assert(m_meshes.size() == frame.index());
-            m_meshes.emplace_back(std::make_unique<IndexedMesh>(frame, vertices, indices));
+            assert(frame.index() < frameCount());
```

Comment:
Just want to confirm this is deliberate? Wasn't expecting changes here in this PR

Reply:
There was a bit of back and forth due to merge conflicts, but I just confirmed that this is the correct change.